L1Trigger/L1TNtuples/interface/L1AnalysisRecoMuon2.h

```diff
@@ -63,8 +63,11 @@ namespace L1Analysis {
   private:
     L1AnalysisRecoMuon2DataFormat recoMuon_;
 
-    PropagateToMuon muPropagator1st_;
-    PropagateToMuon muPropagator2nd_;
+    PropagateToMuonInterface muPropagatorInt1st_;
+    PropagateToMuonInterface muPropagatorInt2nd_;
```

Comment:
The "interface" might be made `const` (did not test this).

Reply:
Yes, done.